charts/postgresql/templates/_helpers.yaml

```diff
@@ -0,0 +1,7 @@
+{{- define "astronomer_bootstrap_url" -}}
```

Comment:
@andriisoldatenko the rest of the umbrella uses a dot pattern, maybe rename to `astronomer.boostrap`?